test-suite/src/function/abs.rs

```diff
@@ -0,0 +1,95 @@
+use crate::*;
+
+test_case!(abs, async move {
+    use gluesql_core::{
+        executor::EvaluateError, executor::Payload, prelude::Value::*, translate::TranslateError,
+    };
+    let test_cases = vec![
+        (
+            "CREATE TABLE SingleItem (id INTEGER)",
+            Ok(Payload::Create),
+        ),
+        (
+            r#"INSERT INTO SingleItem VALUES (0)"#,
+            Ok(Payload::Insert(1)),
+        ),
+        (
+            "SELECT abs(1) AS abs1, 
+                    abs(-1) AS abs2, 
+                    abs(+1) AS abs3 
```

Comment:
sql is case-insensitive, so there's no problem to execute this statement.
However we used uppercase letters for all other keyword `SELECT`, `AS` and `FROM` only except for `abs` which is also the same keyword like others.
It looks better to use uppercase letters for all keyword for the consistency.